### PR TITLE
Added in not eligible view

### PR DESCRIPTION
**Pull request by Daveo-G**

### CODE REVIEW

File: app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@*
```
Comment (ChrisHillCap) [inline app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html]: could this view be named _ kebab case like all the views to keep it consistent
not_eligible_for..

File: app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers.form
+@import core.views.html.helpers._
+@import core.config.AppConfig
+@import core.views.html.templates.main_template
+
+@(postAction: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```
Comment (ChrisHillCap) [inline app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html]: does this need to be a post action, we can pass it in, but it should be just a getAction? i think? theres no form required

Comment (ChrisHillCap) [reply app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html]: edit: if this parameter is not used then it is not needed, so we can just pass the implicits in and nothing else

Comment (Daveo-G) [reply app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html]: Done

File: app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html (hunk 1 above)
Comment (Daveo-G) [reply app/incometax/eligibility/views/NotEligibleForIncomeTax.scala.html]: Done

**Pull request merged**